### ISSUE TITLE
torch backend min vs max behaviour

### ISSUE THREAD

**Issue by jerodway** [keras-team/keras#19064]
When I was trying to use keras.ops.min and keras.ops.max I found a difference with how the axis argument was handled when using the torch backend.

The max operation handles things this way:
`
if axis is None:
      result = torch.max(x)
else:
      result = amax(x, axis=axis, keepdims=keepdims)
`

while the min handles things this way:
`
if axis is None:
    result = torch.min(x)
else:
    if isinstance(axis, list):
        axis = axis[-1]
    result = torch.min(x, dim=axis, keepdim=keepdims)
`

For the min case, only one of the axes are reduced, so using it as a reduce_min replacement is more difficult.  I think that the handling in the max function is more what I'm personally after, but I thought I would check if it the axis handling in the min function is what's intended before sending a pull request.

**Comment by SuryanarayanaY** [comment]
Hi @jerodway ,

Thanks for reporting the issue. Yes the behaviour of min function is not correct with torch backend. Proposed a PR for same.

**Comment by SuryanarayanaY** [comment]
Hi @jerodway ,

The proposed PR got merged and can now work fine with nightly. Thanks for your time in reporting that helps in fixing this bug. Can we mark it as completed now ?
